src/auto/enums.rs

```diff
@@ -14,6 +14,7 @@ pub enum ChecksumType {
     Sha256,
     #[cfg(feature = "v2_36")]
     Sha512,
+    Sha384,
```

Comment:
This is missing a version attribute

Reply:
I didn't notice this one... Thanks for finding it!

Reply:
Thanks, @sdroege, I forgot about this